test_utils/test_translate_wav.bats

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bats
+# -*- mode:sh -*-
+
+setup() {
+    tmpdir=/tmp/espnet-test-st
+}
+
+
+@test "translate_wav" {
+    cd ./egs/fisher_callhome_spanish/st1/
+    wav=../../../test_utils/st_test.wav
+
+    # TODO(karita): why tar results in error?
+    ../../../utils/translate_wav.sh --stop-stage 2 --decode_dir $tmpdir --models fisher_callhome_spanish.transformer.v1.es-en ${wav}
```

Comment:
@hirofumi0810 Do you have smaller fisher_callhome_spanish ST models (e.g., RNN)? That would be great for testing because this model costs 100MB to download.

Reply:
Maybe CI does not matter that in `ci/test_integration.sh`
- before this PR (1e08702): 07:08 min https://circleci.com/gh/espnet/espnet/11291?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link 
- after this PR (1c2ae27): 06:44 min https://circleci.com/gh/espnet/espnet/11317?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link